src/GraphQl/Resolver/Factory/ItemMutationResolverFactory.php

```diff
@@ -87,6 +87,13 @@ public function __invoke(string $resourceClass = null, string $rootClass = null,
 
             $resourceMetadata = $this->resourceMetadataFactory->create($resourceClass);
             $this->canAccess($this->resourceAccessChecker, $resourceMetadata, $resourceClass, $info, $item, $operationName);
+            $dtoClass = $resourceMetadata->getAttribute('input_class');
```

Comment:
This will erase `$resourceClass` whatever value `input_class` has (false, null, empty string...), which will introduce a bug on later lines 101 & 103 where `$resourceClass` is used.